controllers/putContact.js

```diff
@@ -0,0 +1,36 @@
+import { contactSchema } from '../validators/ContactSchema.js';
+import * as contactsActions from '../models/index.js';
+
+export const putContact = async (req, res, next) => {
+    try {
+        const { contactId } = req.params
+
+        if (Object.keys(req.body).length === 0) {
```

Comment:
Jeśli w joi zmienisz wszystko na required to nie trzeba będzie tego sprawdzać, bo joi sprawdzi czy pole jest obecne.

Reply:
Dodałem required () do joi ale zostawiłem
const { value, error } = contactSchema.validate(req.body) - jako validację - tylko nie jestem pewien czy ok.

Może powinienm zrobić tak:
Joi.validate(name/email/phone, schema, (err, val) => {
  if (err) {
    console.log('Walidacja się nie udała!');
  } else {
    console.log('Wszystko w porządku.');
  }
});